bazel/hiredis.bzl

```diff
@@ -0,0 +1,23 @@
+hiredis_build_rule = """
+cc_library(
+    name = "hiredis",
+    srcs = [
+        "async.c",
+        "hiredis.c",
+        "net.c",
+        "read.c",
+        "sds.c",
+    ],
+    hdrs = [
+        "dict.c",
```

Comment:
So first I was wondering: should this be declared in `srcs` instead of `hdrs`?
But after digging in, I think you made the right choice here.

Reply:
added comments to bazel file